features/vat-replacement/run.js

```diff
@@ -175,36 +170,22 @@ function printPreviewModeWarning() {
     console.log('********************************************************************')
 }
 
-function initConfigOptions() {
-    const args = process.argv.slice(2);
-    const configOptions = {
-        ctp : {
-            projectKey : process.env.PROJECT_KEY,
-            clientId : process.env.CLIENT_ID,
-            clientSecret : process.env.CLIENT_SECRET
-        },
-        dryRun: true
-    }
-
-    // Check environment variables
-    if (!configOptions.ctp.projectKey || !configOptions.ctp.clientId || !configOptions.ctp.clientSecret) {
-        console.error('Please set project key, client ID and client secret in environment variables')
+(async function main() {
+    const configOptions = initConfigOptions()
+    if (!configOptions) {
         return
     }
 
+    const args = process.argv.slice(2);
+
     // Check update mode / preview mode
     if (args.length>0 && args[0] === '-update') {
         configOptions.dryRun = false
+    } else {
+        configOptions.dryRun = true
```

Comment:
Do we still need it if default value of `dryRun` property is true ?

Reply:
Where is it defined as default and true? Unassigned variable by default is false.

Reply:
It is used to be here
https://github.com/commercetools/commercetools-cli-scripts/pull/6/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-cce7374c4a7093be4c412239ad28af7636e01383b80f576500ede41c33ec53abL186

I just saw it is now unassigned. It's fine for me.

Reply:
I must have deleted it since `initConfigOptions` is a useful function for all the scripts (moved to the shared utils) and not all of them need `dryRun` option.